<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QDialog" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="text_browser">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>461</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deposit_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>存款</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_balance_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询余额</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transfer_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdraw_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取款</string>
   </property>
  </widget>
  <widget class="QPushButton" name="print_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打印凭条</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quit_button</sender>
   <signal>clicked()</signal>
   <receiver>main_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
